ldclient/client.py

```diff
@@ -172,6 +172,7 @@ class LDClient:
 
     Client instances are thread-safe.
     """
+
```

Comment:
I did add some small whitespace fixes in this PR in addition to the thread renames. I figured it was small enough it wouldn't be a big deal.